src/wasm/wrt/ngx_wrt_wasmer.c

```diff
@@ -558,7 +560,9 @@ ngx_wasmer_init_instance(ngx_wrt_instance_t *instance, ngx_wrt_store_t *store,
                     instance->env.data[i] = (wasm_extern_t *)
                         wasmer_named_extern_unwrap(
                             instance->wasi_imports.data[j]);
+#if (NGX_DEBUG)
```

Comment:
@thibaultcha is there supposed to be an empty line following these `#endif`s?

Reply:
If the surrounding code is already "airy", then usually I keep it that style. If it is dense then I keeping them dense...